CLAUDE.md

```
@@ -1,6 +1,14 @@
-# CLAUDE.md
+# AnthropicLB
 
-This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
+Usage-aware load balancing proxy for Claude Code
+
+### Anthropic Rate Limit Windows (Unified Headers)
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)